README.md

```diff
@@ -47,15 +47,16 @@ To compile and run the Zilliqa codebase, you will need the following dependencie
 * `CMake`
 * `build-essential`
 * `pkg-config` 
+* `json-rpc-cpp`
 
 For Ubuntu 16.04, you can use the following command to install the dependencies:  
 ```bash
-sudo apt-get install libboost-all-dev libssl-dev libleveldb-dev libjsoncpp-dev libsnappy-dev cmake build-essential pkg-config
+sudo apt-get install libboost-all-dev libssl-dev libleveldb-dev libjsoncpp-dev libsnappy-dev cmake libjsonrpccpp-dev build-essential pkg-config
```

Comment:
@sidthekidder just a minor question here: don't we need `libmicrohttpd-dev` here as well? `libmicrohttpd-dev` seems not existing on a fresh Ubuntu 16.04.

Reply:
updated 👍